docs/user/dashboard/lens.asciidoc

```diff
@@ -32,15 +36,23 @@ If you're unsure about the visualization type you want to use, or how you want t
 
 If you already know the visualization type you want to use, and how you want to display the data, use the following process:
 
-* *Choose the visualization type.* Open the *Chart type* dropdown, then select the visualization type before you drag any fields.  
-+
-To view more visualizations that *Lens* automatically created for the fields, click the *Suggestions*. If one of the *Suggestions* meets your visualization needs, click *Save and return* to add it to the dashboard.
+*Choose the visualization type.* 
 
-* *Choose the data you want to visualize.* Drag the fields directly to the layer pane. *Lens* automatically selects the aggregation function.
-+
-If you want to learn more about the data a field contains, click *i* next to the field.
+. Before you drag fields to the workspace, open the *Chart type* dropdown, then select the visualization type.  
+
+. To view more visualizations that *Lens* automatically created for the fields, click the *Suggestions*. If one of the *Suggestions* meets your visualization needs, click *Save and return* to add it to the dashboard.
+
+*Choose the data you want to visualize.* 
 
-* *Edit and delete.* To change the *Quick function* and display options, click the field in the layer pane. To delete a field, close the configuration options, then click *X* next to the field.
+. Drag the fields directly to the layer pane. *Lens* automatically selects the aggregation function.
+
+. If you want to learn more about the data a field contains, click *i* next to the field.
```

Comment:
Remove "If you want"?

The "i" is a little hard to parse.  What about including the icon instead?